packages/babel-helper-member-expression-to-functions/src/index.js

```diff
@@ -167,13 +167,19 @@ const handle = {
       const parentIsOptionalCall = parentPath.isOptionalCallExpression({
         callee: node,
       });
+      const isParenthesizedMemberCall =
+        parentPath.isCallExpression({ callee: node }) &&
+        node.extra?.parenthesized;
```

Comment:
The current approach does not support `parserOpts.createParenthesizedExpressions`. When developing I realize that the whole transform does not take `ParenthesizedExpressions` into account, I suggest we address it in a separate PR before this one gets too big to review.